CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Fixed
 - Permissions - Fix automated permission for projects having multiple teams.
+- Namespacing - Fix conflict with ResourcesController
```

Comment:
You added the change in a released version (v2.8.4) but your fixed will be available in new gem version.
Can you create a new "Fixed" section just below "[Unreleased]" with your change description, like this:

 ```markdown
## [Unreleased]
### Fixed
- Namespacing - Fix conflict with ResourcesController
```